README.md

```diff
@@ -46,3 +46,5 @@ Simply download and install the archive file for the version you want and unpack
 For example, if you download [scalabrad-0.3.1.tar.gz](https://bintray.com/artifact/download/labrad/generic/scalabrad-0.3.1.tar.gz) and untar it in some directory `$PATH/`, then you can run `$PATH/scalabrad-0.3.1/bin/labrad` to start the manager. You'll need to have Java 8 installed on your system, but all other dependencies are included in the package. Windows `.bat` files are also included in the distribution, though you may need a tool like 7-Zip to extract the `.tar.gz` archive.
```

Comment:
While you're in here making some changes, maybe update these links to point to 0.5.0 instead of 0.3.1.